server/base/src/main/javadoc/META-INF/NOTICE

```diff
@@ -1,5 +0,0 @@
-Apache Accumulo
-Copyright 2011-2016 The Apache Software Foundation
```

Comment:
They might not have been in earlier versions of the build plugins... but that wouldn't explain why we didn't add this to the other modules. I gave up trying to understand, and just fixed it. :)